### PR TITLE
Added known issue in the frontend and added release notes for the ass…

### PR DESCRIPTION
**Pull request by Rick-Ritense**
…ignee in the backedn

### CODE REVIEW

**Review by mritense — APPROVED** [review]

File: release-notes/major9/09.23.0/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -22,6 +22,26 @@ The following features were added:
    }
   ```
 
+* **A document can now be assigned to a user**
+
+  There are two methods of assigning a user to a document.
+  
+  1) Using the endpoint `POST /api/document/{documentId}/assign` and providing the assignee id in the request body.
```
Comment (marijnritense) [inline release-notes/major9/09.23.0/valtimo-backend-libraries.md:29]: I think this should go on a different page. The release notes target more than just developers, so try and keep the description here as functional as possible. And for more details, link to a corresponding using valtimo/extending valtimo page.

File: extending-valtimo/document/assigning-a-user.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Assigning a user
+
+It is possible to assign a user to a document. There are two methods available
```
Comment (marijnritense) [inline extending-valtimo/document/assigning-a-user.md:3]: Missing a short introduction here. Something like https://github.com/valtimo-platform/valtimo-documentation/blob/main/using-valtimo/document/create-document-definition.md
Maybe you can copy the first part of that introduction? So it becomes something like:

A document definition is a JSON document that defines the structure that document has, as well as restrict the allowed values for each property through validation rules. This page shows how to assign and unassign users from and to a case.

## Assigning a user
There are two ways to assign a user to a case:
...

## Unassigning a user
...

File: extending-valtimo/document/assigning-a-user.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Assigning a user
```
Comment (marijnritense) [inline extending-valtimo/document/assigning-a-user.md:1]: Assigning and unassigning a user

File: using-valtimo/document/assigning-a-user.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Assigning a user to a document
+
+Every document can now be assigned to a user.
```
Comment (marijnritense) [inline using-valtimo/document/assigning-a-user.md:3]: This needs an introduction as well

File: using-valtimo/document/assigning-a-user.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+It is possible to assign a user to the current process by creating a service task and add the following expression
+
+```
+${documentDelegate.setAssignee(execution, "email address of user")}
+```
+
+When this service task is reached the assignee will be set
+
+![Assign user in process](img/assign-user-in-process.png)
+
```
Comment (marijnritense) [inline using-valtimo/document/assigning-a-user.md:27]: What about unassigning?

Comment (Rick-Ritense) [reply using-valtimo/document/assigning-a-user.md:27]: This is not yet available in the user interface. But I'll add a commented sections that can be enabled in the future when the feature becomes available

**Pull request merged**